build.gradle

```diff
@@ -274,7 +274,7 @@ String bwcFilePath = "src/test/resources/org/opensearch/ad/bwc/"
 testClusters {
     "${baseName}" {
         testDistribution = "ARCHIVE"
-        versions = ["7.10.2","1.0.0"]
+        versions = ["7.10.2","1.1.0-SNAPSHOT"]
```

Comment:
Should we also test upgrade with last version OS 1.0?

Reply:
Hi @ylwu-amzn, the corresponding bwc tests are added so that the core framework for bwc for plugins is ready to use and they are an example of how the bwc tests can be added in the plugins. Each plugin owners can now take up the tests and enhance them to include multiple versions and test required functionalities.

For the CI failure, I am working on fixing that since AD is consuming the snapshot build while `job-scheduler` and `common-utils` are not, I am making corresponding changes in the respective repos.